contracts/GoodGhosting_Polygon.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.6.11;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Pausable.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./aave/ILendingPoolAddressesProvider.sol";
+import "./aave/ILendingPool.sol";
+import "./aave/AToken.sol";
+import "./aave/IncentiveController.sol";
+import "./quickswap/IRouter.sol";
+import "./GoodGhostingWhitelisted.sol";
+import "./GoodGhosting.sol";
+
+/**
+ * Play the save game.
+ *
+ */
+
+contract GoodGhosting_Polygon is Ownable, Pausable, GoodGhosting {
```

Comment:
GoodGhosting is already pausable and ownable, so no need to define them again.

```suggestion
contract GoodGhosting_Polygon is GoodGhosting {
```

Reply:
my bad missed this

Reply:
no worries ;)